packages/flutter/lib/src/widgets/autofill.dart

```diff
@@ -0,0 +1,181 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+import 'framework.dart';
+
+export 'package:flutter/services.dart' show AutofillHints;
+
+/// An [AutofillScope] widget that defines a group of [AutofillClient]s.
+///
+/// {@macro flutter.services.autofill.AutofillScope}
```

Comment:
Before explaining how this class relates to AutofillClient it would be helpful to explain what it's for. Which, as I understand it, is to define a group of widgets that will be autofilled together and which must be built together.